<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">QTableView::item {
    border-bottom: 1px solid white;
}

QTableWidget::item { 
border-right: 1px solid white; }</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>сохранить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="usl">
     <property name="text">
      <string>условие</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
